modules/apps/layout/layout-admin-web/src/main/java/com/liferay/layout/admin/web/internal/portlet/action/EditLayoutMVCActionCommand.java

```diff
@@ -149,10 +149,25 @@ protected void doProcessAction(
 			actionRequest, "useCustomCanonicalURL");
 		Map<Locale, String> canonicalURLMap =
 			LocalizationUtil.getLocalizationMap(actionRequest, "canonicalURL");
+		boolean useCustomDescription = ParamUtil.getBoolean(
+			actionRequest, "useCustomDescription");
+		Map<Locale, String> openGraphDescriptionMap =
+			LocalizationUtil.getLocalizationMap(
+				actionRequest, "openGraphDescription");
+		boolean useCustomTitle = ParamUtil.getBoolean(
+			actionRequest, "useCustomTitle");
+		Map<Locale, String> openGraphTitleMap =
+			LocalizationUtil.getLocalizationMap(
+				actionRequest, "openGraphTitle");
+
+		long openGraphImageFileEntryId = ParamUtil.getLong(
+			actionRequest, "openGraphImageFileEntryId");
 
 		_layoutSEOEntryService.updateLayoutSEOEntry(
 			groupId, privateLayout, layoutId, useCustomCanonicalURL,
-			canonicalURLMap, serviceContext);
+			canonicalURLMap, useCustomDescription, openGraphDescriptionMap,
+			useCustomTitle, openGraphTitleMap, openGraphImageFileEntryId,
+			serviceContext);
```

Comment:
Look for "String title" in the portal. In method signatures, the title always comes immediately before than the description, so we should do the same here. E.g. doFoo(useDescription, descriptionMap, useTitle, titleMap) ==> doFoo(useTitle, titleMap, useDescription, description)